packages/server/src/automations/utils.ts

```diff
@@ -111,10 +117,16 @@ export async function updateTestHistory(
   )
 }
 
-export function removeDeprecated(definitions: any) {
+export function removeInvalidDefinitions(
+  definitions: Record<string, AutomationStepSchema>
+) {
+  const disallowedStepIds: (
```

Comment:
Should this not be handled on the frontend? We just want not to be able to add these types, but they should be part of the definitions. These are used for automation logs and similar, so they should be there

Reply:
For now I think they should be fully removed - since row actions isn't released yet I think its best if there's no mention of it at all.

Reply:
Discussed - definitions are used for automation history as well, going to leave automation definitions full for now instead, logic for disabling creation is on frontend.